tests/helpers/test_trigger.py

```diff
@@ -197,3 +199,58 @@ def non_cb_action(*args):
 
         log_cb.reset_mock()
         unsub()
+
+
+async def test_pluggable_action(hass: HomeAssistant, calls: list[ServiceCall]):
```

Comment:
I think we should test more than one plug and more than one action at a time as there's logic for the relationship between triggers, plugs and actions.

Reply:
I've added some steps with more actions active.